src/main/java/eu/xenit/alfred/initializr/generator/extensions/zipkin/ZipkinDockerComposeCustomizer.java

```diff
@@ -0,0 +1,21 @@
+package eu.xenit.alfred.initializr.generator.extensions.zipkin;
+
+import eu.xenit.alfred.initializr.generator.docker.compose.DockerCompose;
+import eu.xenit.alfred.initializr.generator.docker.compose.DockerComposeCustomizer;
+import eu.xenit.alfred.initializr.model.docker.ComposeServices;
+import eu.xenit.alfred.initializr.model.docker.DockerComposeModel;
+
+public class ZipkinDockerComposeCustomizer implements DockerComposeCustomizer {
+
+    @Override
+    public void customize(DockerCompose compose) {
+        DockerComposeModel zipkinModel = compose.file("zipkin");
+        ComposeServices services = zipkinModel.getServices();
```

Comment:
Alfresco doesn't need any configuration ?

Reply:
It does, see the customizer for alfresco-zipkin-repo